.github/workflows/linting.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Code Linting
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+  test_code:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 12
+    steps:
+      - uses: actions/checkout@v3
+      - name: Lint with flake8
+        run: |
+          git submodule update --init --recursive
```

Comment:
you should be able add:
```
with:
  submodules: 'recursive'
```
to `actions/checkout@v3` to get the same effect, see https://github.com/actions/checkout#usage

Reply:
@proppy Also this?